docs/spec/spec.md

```diff
@@ -1851,15 +1857,52 @@ service on new graphql:Listener(9090) {
 
 ##### 7.1.9.1 The `enabled` Field
 
-The optional field `enabled` accepts a `boolean` that denotes whether the server-side operation cache is enabled or not. By default, it has been set to `true`.
+The field `enabled` accepts a `boolean` that denotes whether the server-side operation cache is enabled or not. By default, it has been set to `true`.
 
 ##### 7.1.9.2 The `maxAge` Field
 
-The optional field `maxAge` accepts a valid `decimal` value which is considered as the TTL(Time To Live) in seconds. The default maxAge is `60` seconds.
+The field `maxAge` accepts a valid `decimal` value which is considered as the TTL(Time To Live) in seconds. The default maxAge is `60` seconds.
 
 ##### 7.1.9.3 The `maxSize` Field
 
-The optional field `maxSize` accepts an int that denotes the maximum number of cache entries in the cache table. By default, it has been set to `120`.
+The field `maxSize` accepts an int that denotes the maximum number of cache entries in the cache table. By default, it has been set to `120`.
+
+#### 7.1.10 Document Cache Configurations
+
+The `documentCacheConfig` field is used to provide the document cache configuration to enable the [GraphQL document caching](#1072-document-caching).
+
+###### Example: Enable Document Cache with Default Values
+
+```ballerina
+@graphql:ServiceConfig {
+    documentCacheConfig: {}
+}
+service on new graphql:Listener(9090) {
+    // ...
+}
+```
+
+###### Example: Document Cache Configurations
+
+```ballerina
+@graphql:ServiceConfig {
+    documentCacheConfig: {
+        enabled: true
+        maxSize: 50
+    }
+}
+service on new graphql:Listener(9090) {
+    // ...
+}
+```
+
+##### 7.1.10.1 The `enabled` Field
+
+The field `enabled` accepts a `boolean` that denotes whether the document cache is enabled or not. By default, it has been set to `true`.
```

Comment:
Maybe we can add a note here stating that even though this value is set to `true`, the document caching is not enabled if the `documentCacheConfig` field is set in the `ServiceConfig` annotation.